src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -0,0 +1,337 @@
+---
+title: "Session replay"
+metaDescription: "Play back user interactions to debug faster and improve page performance."
+---
+
+import browserSessionReplayDetails from 'images/browser_screenshot-full_session-replay-details.webp'
+
+import browserTextMaskingExample from 'images/browser_screenshot-full_text-masking-example.webp'
+
+import browserApplicationSettingsPage from 'images/browser_screenshot-full_Application-settings-page.webp'
+
+import browserMaskSettings from 'images/browser_screenshot-full_mask-settings.webp'
+
+import browserSessionReplayAppSettings from 'images/browser_screenshot-full_session-replay-app-settings.webp'
+
+import browserSessionReplayPage from 'images/browser_screenshot-full_Session-replay-page.webp'
+
+import browserErrorsInboxSessionReplay from 'images/browser_screenshot-full_errors-inbox-session-replay.webp'
+
+<Callout variant="important">
+  Session replay is in limited preview. While not everyone can access it just yet, you can request access to the preview [here](https://newrelic.com/platform/session-replay-early-access).
+</Callout>
+
+Session replay allows you to play back user interactions on your web app and map other telemetry data to help you understand your user's journeys. Session replay can be used for troubleshooting and improving the end user experience, such as:
+
+* **Troubleshooting JavaScript errors**: Troubleshoot the cause of an error by showing you what the user was doing when the error occurred. For example, if an end user receives an error while trying to complete an ecommerce transaction, you can use session replay to see which steps they took and what data they entered before the error occurred. This can help you quickly identify and fix the root cause of the problem.
```

Comment:
```suggestion
* **Troubleshooting JavaScript errors**: Troubleshoot the cause of an error by seeing what the user was doing when the error occurred. For example, if an end user receives an error while trying to complete an ecommerce transaction, you can use session replay to see which steps they took and what data they entered before the error occurred. This can help you quickly identify and fix the root cause of the problem.
```